cmake/Config.cmake

```diff
@@ -44,7 +44,20 @@ endif()
 # -DPython_ADDITIONAL_VERSIONS=Major.Minor
 # if your python version is not implicitly supported by cmake
 find_package (PythonInterp 2.7)
-find_package (PythonLibs 2.7)
```

Comment:
We need it on Windows because python-config doesn't appear to be available. It produces the correct answer on Linux.